packages/pyright-internal/src/localization/simplified.nls.de.json

```diff
@@ -0,0 +1,93 @@
+        "argMorePositionalExpectedOne": "1 weiteres positionelles Argument erwartet.",
+        "argPositional": "Positionelles Argument erwartet",
+        "argPositionalExpectedOne": "1 positionelles Argument erwartet.",
+        "assignmentTargetExpr": "Ein Ausdruck kann kein Zuweisungsziel sein",
+        "booleanIsLowerCase": "\"{name}\" ist nicht definiert, meinten Sie \"{booleanName}\"?",
+        "breakOutsideLoop": "\"break\" kann nur innerhalb einer \"while\" oder \"for\" Schleife verwendet werden",
+        "constructorNoArgs": "Keine Argumente für \"{type}\" Konstruktor erwartet",
+        "continueOutsideLoop": "\"continue\" kann nur innerhalb einer \"while\" oder \"for\" Schleife verwendet werden",
+        "delTargetExpr": "Ausdruck kann nicht gelöscht werden.",
+        "duplicateParam": "Duplizierter Parameter \"{name}\"",
+        "expectedAssignRightHandExpr": "Auf der rechten Seite von \"=\" wird ein Ausdruck erwartet",
+        "expectedBinaryRightHandExpr": "Auf der rechten Seite vom Operator wird ein Ausdruck erwartet",
+        "expectedClassName": "Klassenname fehlt",
+        "expectedCloseBrace": "Schließende geschweifte Klammer „}“ fehlt.",
+        "expectedCloseBracket": "Schließende eckige Klammer „]“ fehlt.",
+        "expectedCloseParen": "Schließende runde Klammer „)“ fehlt.",
+        "expectedColon": "Doppelpunkt „:“ fehlt",
+        "expectedDelExpr": "Ausdruck nach „del“ fehlt.",
+        "expectedElse": "„else“ fehlt",
+        "expectedEqualityOperator": "Hier wird ein Gleichheitsoperator erwartet, meinten Sie \"==\"?",
+        "expectedExpr": "Ausdruck fehlt",
+        "expectedFunctionName": "Funktionsname nach \"def\" fehlt.",
+        "expectedIdentifier": "Bezeichner fehlt",
+        "expectedImport": "„import“ fehlt",
+        "expectedImportAlias": "Fehlendes Symbol nach \"as\"",
+        "expectedImportSymbols": "Nach dem Import wird ein oder mehrere Symbolnamen erwartet",
+        "expectedIn": "„in“ fehlt.",
+        "expectedIndentedBlock": "Einrückung fehlt",
+        "expectedInExpr": "Fehlender Ausdruck nach \"in\"",
+        "expectedMemberName": "Fehlender Mitgliedsname nach \".\"",
+        "expectedModuleName": "Modulname fehlt",
+        "expectedNamedParameter": "Schlüsselwortparameter muss \"*\" folgen",
+        "expectedNewlineOrSemicolon": "Unerwartete zusätzliche Inhalte\nEs wird eine Anweisung pro Zeile erwartet",
+        "expectedOpenParen": "Öffnende runde Klammer „)“ fehlt.",
+        "expectedParamName": "Fehlender Parametername",
+        "expectedSliceIndex": "Fehlender Index oder Slice",
+        "expectedYieldExpr": "Ein Ausdruck wird in der yield-Anweisung erwartet",
+        "globalReassignment": "\"{name}\" wird vor der globalen Deklaration zugewiesen",
+        "globalRedefinition": "\"{name}\" wurde bereits als global deklariert",
+        "importResolveFailure": "Modul „{importName}“ konnte nicht gefunden werden.",
```

Comment:
The space after Modul is `U+00a0`. Flagged in VSCode, presumably no impact.

Reply:
I may as well fix these